components/renku_data_services/migrations/versions/1ef98b967767_add_command_and_args_to_environment.py

```diff
@@ -0,0 +1,54 @@
+"""Add command and args to environment
+
+Revision ID: 1ef98b967767
+Revises: 584598f3b769
+Create Date: 2024-08-25 21:05:02.158021
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision = "1ef98b967767"
+down_revision = "584598f3b769"
+branch_labels = None
+depends_on = None
+
+default_command = ["sh", "-c"]
+default_args = [
+    "jupyter server --ServerApp.ip=0.0.0.0 --ServerApp.port=8888 --ServerApp.base_url=$RENKU_BASE_URL_PATH "
+    '--ServerApp.token="" --ServerApp.password="" --ServerApp.allow_remote_access=true '
+    "--ContentsManager.allow_hidden=true --ServerApp.allow_origin=*",
+]
```

Comment:
See: https://renku-ci-ds-517.dev.renku.ch/v2/projects/flora.thiebaut/flora-test.

This runs the `/post-init.sh` script if present.

```suggestion
default_args = [
    "/entrypoint.sh  jupyter server --ServerApp.ip=0.0.0.0 --ServerApp.port=8888 --ServerApp.base_url=$RENKU_BASE_URL_PATH "
    '--ServerApp.token="" --ServerApp.password="" --ServerApp.allow_remote_access=true '
    "--ContentsManager.allow_hidden=true --ServerApp.allow_origin=*",
]
```